src/App.test.js

```diff
@@ -1,9 +1,7 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
 import App from './App';
+import './setupTests.js'
 
 it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+  const wrapper = shallow(<App />);
+  wrapper.unmount();
```

Comment:
It is testing. Basically it will error out if it fails to render. So there's no real 'expect' per se. It's more of a don't have errors thing. This is generated by React so feel comfortable assuming they don't explicitly test for no errors.

Reply:
Cool